### DISCUSSION TITLE
CasaOS VM instead of LXC for easy USB passthrough

### DISCUSSION THREAD

**Discussion by gadiyabhushan** [tteck/Proxmox#968] [Help Needed] [answered]
I have installed CasaOS as an LXC using the script here. I have set up Radarr/Prowler/QbitTorrent etc on this. Now I want to attach an external HD to this LXC but it is very difficult. I searched a lot for this but cannot get a solution. I can see that if it would have been a VM, mounting a USB device is very easy. Can we have a VM for CasaOS?

**Comment by tteck** [comment, marked as answer]
Just added:
If the LXC is created Privileged, the script will automatically set up USB passthrough

**Reply by gadiyabhushan** [reply]
How do you create an Privileged CasaOS LXC?

**Reply by tteck** [reply]
"Advanced"
